### ISSUE TITLE
Prevent Duplicate Orders/Update Existing Order

### ISSUE THREAD

**Issue by scottfriebel** [printdotio/printio-api#42]
Is there a way, when submitting an order via the api, to specify to your system to not allow multiple orders with the same “SourceId” or “Partner Order Item ID”? Or that if an order already exists with the ID being submitting that the info being submitted be used to update the existing order, instead of a create a new order with the same ID?

**Comment by micahasmith** [comment]
@scottfriebel no, but you can turn on "Manual Approval" which means orders need to be manually approved in the admin panel. If you did this then you could manually keep duplicates out.
